Source/WebKit/UIProcess/Inspector/WebPageInspectorController.cpp

```diff
@@ -96,7 +96,7 @@ void WebPageInspectorController::connectFrontend(Inspector::FrontendChannel& fro
     m_frontendRouter->connectFrontend(frontendChannel);
 
     if (connectingFirstFrontend)
-        m_agents.didCreateFrontendAndBackend(&m_frontendRouter.get(), &m_backendDispatcher.get());
+        m_agents.didCreateFrontendAndBackend(m_frontendRouter.ptr(), m_backendDispatcher.ptr());
```

Comment:
It seems `didCreateFrontendAndBackend` implementations do not use these two parameters, maybe we could remove them (not in this patch though).